02 - Community Expansions/02.05 All Community Expansions/CSS Snippets/Image Adjustments.md

```diff
@@ -26,8 +26,10 @@ Included in the [[ITS Theme]].
 ##### How to use the snippet:
 [Readme Guide](https://github.com/SlRvb/Obsidian--ITS-Theme#image-positions)
 
-**Video Explanation:** ["Slides" from the video](https://slrvb.github.io/Site/3_Nebula/ITS-Theme/ITST_Image-Positions/)
-<iframe width="100%" height="250px" src="https://www.youtube.com/embed/VRoBNWvw8sU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+**Video Explanation:**
```

Comment:
but that's not a video though :D

Reply:
It is when you open the note 😉
![Obsidian_cQIfQDysjW](https://user-images.githubusercontent.com/54087190/146635842-a6d598fc-94d5-4008-aaa5-3dcdcbf6ff20.png)

Reply:
ah right, I always forgot about these embedded block references, my bad, sorry! 🙈